src/sentry/tasks/post_process.py

```diff
@@ -1455,6 +1455,40 @@ def should_postprocess_feedback(job: PostProcessJob) -> bool:
     return False
 
 
+def check_has_high_priority_alerts(job: PostProcessJob) -> None:
+    """
+    Determine if we should fire a task to check if the new issue
+    threshold has been met to enable high priority alerts.
+    """
+    try:
```

Comment:
Nit: I'd recommend just putting this try/except around the `lock.acquire` section